MapboxCoreNavigation.podspec

```diff
@@ -40,7 +40,7 @@ Pod::Spec.new do |s|
   s.requires_arc = true
   s.module_name = "MapboxCoreNavigation"
 
-  s.dependency "MapboxDirections.swift", "~> 0.21.0"
+  s.dependency "MapboxDirections.swift", :git => "https://github.com/mapbox/MapboxDirections.swift.git", :branch => "then"
```

Comment:
Unfortunately, something like this does not work. Only published pods are able to be pulled in as deps here.

Reply:
😞